src/__tests__/example.js

```diff
@@ -8,7 +8,8 @@ import {COMMON} from '../constants'
 // 2. remove this definition; it's just for eslint
 function SomeComponent() {}
 
-// 3. remove the leading "x" from this line to enable the test
+// 3. remove the ".skip" from this line to enable the test
+// eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Thanks for cleaning this up! I don't mind this approach, but since the code is likely to be copied, I wonder if we can do one of these:

### ignore the rule in eslint config
I'm wondering if we should add this exception at the eslint config level, just because it's likely this code will be copied. You can add this to `src/__tests__/.eslintrc.json`:

```
"overrides": [
    {
      "files": [ "example.js" ],
      "rules": {
        "jest/no-disabled-tests": "off"
      }
    }
  ]
```

### ignore this file in jest config

Another alternative is to just remove the `.skip` in the example file, and ignore this file in the `jest.config.js`:

```
  testPathIgnorePatterns: ['example.js']
```

Of course, if we do this, we can remove the `.skip` altogether.

Reply:
Updated @T-Hugs